chemprop/uncertainty/calibrator.py

```diff
@@ -197,9 +211,17 @@ def fit(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -> Non
 
             scores_j = torch.gather(scores_j, 1, targets_j.unsqueeze(1)).squeeze(1)
             num_data = targets_j.shape[0]
-            q_level = (torch.ceil((num_data + 1) * (1 - self.alpha)) / num_data).item()
+            if self.alpha >= 1 / (num_data + 1):
+                q_level = (math.ceil((num_data + 1) * (1 - self.alpha)) / num_data).item()
```

Comment:
how is `q_level` a tensor here? `num_data` and `self.alpha` are of type `int` and `float`, respectively?